app/src/main/res/values-de/strings.xml

```diff
@@ -77,11 +79,16 @@
         <item quantity="one">%d Tag</item>
         <item quantity="other">%d Tage</item>
     </plurals>
+    <string name="textFieldOptional">(Fakultativ)</string>
+    <string name="transferMessagePlaceholder">Eine Botschaft, die Sie weitergeben möchten?</string>
+    <string name="transferRecipientAddressPlaceholder">Senden Sie die Dateien an</string>
     <string name="transferSendButton">Senden</string>
+    <string name="transferSenderAddressPlaceholder">Ihre E-Mail Adresse</string>
```

Comment:
The strings are conjugated to the polite form and to the 2nd person plural form instead of the 2nd person singular form

Reply:
We don't use polite form in the app ? So uncivilised